src/DonationForms/DataTransferObjects/DonationFormGoalData.php

```diff
@@ -50,6 +52,7 @@ class DonationFormGoalData implements Arrayable
     public $goalEndDate;
 
     /**
+     * @unreleased Campaign is the default goal type
```

Comment:
Can we remove this? I think you rolled back the changes previously made in the constructor but kept the `@unreleased` tag.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7569/commits/9d80e7d9e7b3c3415aac2fc112d2113d0e6773d7